openapi/sms/v1-review/partner-callbacks.yaml

```diff
@@ -0,0 +1,169 @@
+openapi: '3.0.3'
+info:
+  version: 0.0.1
+  title: Provider callbacks
+  description: |
+    An SAP Emarsys SMS Partner Service allows partners to host SMS message sends that are triggered
+    by an SAP Emarsys SMS campaign launch. This OpenAPI specification describes the possible callback endpoints.
+  contact:
+    name: SAP Emarsys SMS Team
+    email: team-sms@emarsys.com
+x-sap-shortText: Process Partner service callbacks for the Emarsys SMS Partner API. Expected callbacks contain outbound message delivery reports and inbound messages.
+servers:
+  - url: https://api.emarsys.net/sms/partner/v1
+paths:
+  /clients/{clientId}/deliveryReports:
+    post:
+      summary: Delivery reports callback
+      operationId: sendDeliveryReports
+      tags: [Delivery Reports]
+      security:
+        - oAuthAuthorization: []
+      parameters:
+        - $ref: './common.yaml#/components/parameters/clientIdPathParam'
+      requestBody:
+        description: The body of the delivery reports in JSON format
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/DeliveryReportBatch'
+      responses:
+        '202':
+          $ref: './common.yaml#/components/responses/AcceptedResponse'
+        '400':
+          $ref: './common.yaml#/components/responses/InvalidFormatResponse'
```

Comment:
Since it's reused schema it's example could be confusing to a customer maybe as it's says `"message": "should have required property 'messageId'"` but in this endpoint request there is no such field.

Reply:
good point, we will change to a place holder `foo`